custom_boot/package/kiwi-boot-descriptions.spec

```diff
@@ -192,6 +192,16 @@ Requires:       jing
 Meta package for the buildservice to pull in all required packages
 for the build host to build simple disk images
 
+%package -n kiwi-image-tbz-requires
+Summary:        KIWI - buildservice host requirements for root archive tarball
+Group:          System/Management
+Provides:       kiwi-image:tbz
+Requires:       kiwi-filesystem-requires
```

Comment:
why do we need filesystem-requires for creating an archive of the root tree ? I thought a tar requirement should be enough ?

Reply:
I have just copied from the other kiwi-image:* packages.  Probably tar + xz should be enough.